### PR TITLE
Allow custom package comments

### PR DESCRIPTION
**Pull request by mosBrkr**
Fixes [#7369](https://github.com/wixtoolset/issues/issues/7369)

### CODE REVIEW

**Comment by mosBrkr** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mosBrkr** [comment]
recheck

**Comment by mosBrkr** [comment]
recheck

File: src/wix/WixToolset.Core/Compiler_Package.cs [hunk 1]
```diff
@@ -55,6 +56,9 @@ private void ParsePackageElement(XElement node)
                     case "Codepage":
                         codepage = this.Core.GetAttributeLocalizableCodePageValue(sourceLineNumbers, attrib);
                         break;
+                    case "Comments":
```
Comment (robmen) [inline src/wix/WixToolset.Core/Compiler_Package.cs:59]: The `Comments` attribute should go on the `SummaryInformation` element, not the `Package` element. This is a change in WiX v4 as the old `Product` element was renamed `Package`, and the old `Package` element was renamed `SummaryInformation`.

**Review by robmen — CHANGES_REQUESTED** [review]

File: src/wix/WixToolset.Core/Compiler_Package.cs (hunk 1 above)
Comment (mosBrkr) [reply src/wix/WixToolset.Core/Compiler_Package.cs]: Sorry I got you wrong. I moved it to the `SummaryInformation` element

File: src/wix/WixToolset.Core/Compiler_Package.cs [hunk 2]
```diff
@@ -190,12 +190,6 @@ private void ParsePackageElement(XElement node)
                     Value = "Installation Database"
                 });
 
-                this.Core.AddSymbol(new SummaryInformationSymbol(sourceLineNumbers)
```
Comment (robmen) [inline src/wix/WixToolset.Core/Compiler_Package.cs:193]: `SummaryInformation` is an optional element. This default value still needs to be set when no `SummaryInformation` is provided.

File: src/wix/test/WixToolsetTest.CoreIntegration/TestData/Comments/Default.wxs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+  <Package Version="1"
+           Name="MsiPackage"
+           Manufacturer="Example Corporation"
+           UpgradeCode="047730a5-30fe-4a62-a520-da9381b8226a">
+    <MajorUpgrade DowngradeErrorMessage="Downgrade not allowed" />
+    <SummaryInformation />
```
Comment (robmen) [inline src/wix/test/WixToolsetTest.CoreIntegration/TestData/Comments/Default.wxs:7]: A `SummaryInformation` without attributes shouldn't be allowed, but not necessary to fix it as part of this change.

**Review by robmen — CHANGES_REQUESTED** [review]

**Comment by robmen** [comment]
PS: Something is up with the CLA bot. I'll investigate that independently.

**Comment by mosBrkr** [comment]
@robmen Can I support you somehow with the CLA bot issue?

**Comment by robmen** [comment]
recheck

**Review by robmen — APPROVED** [review]

**Comment by robmen** [comment]
Thanks. Sorry, I got busy and didn't get this finished earlier. I just had to rebase it one more time then the CLA-Bot finally started passing.

**Pull request merged**